tasks.py

```diff
@@ -195,6 +195,23 @@ def build(ctx, binaries, architectures, registry="quay.io", repo="metallb", tag=
     for arch in architectures:
 
         for bin in binaries:
+            try:
+                if _is_podman():
+                    run ("podman image rm {registry}/{repo}/{bin}:{tag}-{arch}".format(
```

Comment:
since these seems to be identical, can we have the command as a string argument?

something like
command = docker
if podman: 
    command = podman

and then you use it when building the run string.

Reply:
done